content/800-data-platform/250-environments/index.mdx

```diff
@@ -7,21 +7,21 @@ tocDepth: 3
 
 <TopBlock>
 
-For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository.
+For any project in the Prisma Data Platform, you can create multiple environments. You can associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository (if schema syncrhonization from GitHub is enabled).
 
 </TopBlock>
 
 ## About environments in the Prisma Data Platform
 
 You can dedicate each environment to a different stage of your database application life-cycle, such as production, staging, development, or any others that make sense for your project.
 
-For each environment in a [project](/data-platform/projects), you define:
+For each new environment in a [project](/data-platform/projects), you define:
 
-- a GitHub branch (based on the repository of the Production environment)
 - a database connection string
 - a Data Proxy region
+- ([if schema synchronization is enabled](/data-platform/projects/create#optional-enable-schema-synchronization)) a GitHub branch from the repository of the Production environment
```

Comment:
Does this mean you associate the environment with a branch from the same repo as the production environment? If so maybe:
```suggestion
- ([if schema synchronization is enabled](/data-platform/projects/create#optional-enable-schema-synchronization)) a GitHub branch from the same repository as the Production environment
```